top.sls

```diff
@@ -1,11 +1,14 @@
 base:
   '*':
     - apt-common
-#  'E@irill*':
-#    - match: compound
-#    - debile-servers
-#    - debile-servers-users
-  'E@blade*':
+#   - obs-common
+  'E@irill*':
     - match: compound
-    - llvm-slave
-    - apt-common-pinning
+    - debile-servers
+    - debile-servers-users
+#   - mysql_init
+#   - obs-server
```

Comment:
Are you going to also create a category obs-client or obs-worker?

Reply:
Yes I'm planning to do that after this.